tailwind.config.js

```diff
@@ -41,5 +41,4 @@ module.exports = {
       },
     },
   },
-  plugins: [require("@tailwindcss/line-clamp")],
```

Comment:
This is included as of Tailwind v3.3, so we don't need to explicitly include it anymore.